packages/query/src/block-processor.ts

```diff
@@ -2,6 +2,13 @@ import { RootQuerier } from './root-querier';
 import { sha256Hash } from '@penumbra-zone/crypto-web/src/sha256';
 import { computePositionId, getLpNftMetadata } from '@penumbra-zone/wasm/src/dex';
 import { decodeSctRoot } from '@penumbra-zone/wasm/src/sct';
+
+import Array from '@penumbra-zone/polyfills/src/Array.fromAsync';
```

Comment:
What do we need the polyfill for?

Reply:
Whoops — guessing I had been using `Array.fromAsync` before but removed it, but the linter didn't catch it since we're using `Array` in this file. 

I just made another commit to remove this import. Good catch.